saleor/account/utils.py

```diff
@@ -2,6 +2,9 @@
 import random
 
 from django.conf import settings
+from django.contrib.admin.views.decorators import (
+    staff_member_required as _staff_member_required,
+)
```

Comment:
What about naming it `django_staff_member_required` so we explicitly know it is not internal, but django's?

Reply:
Good idea. :)